lib/tessel/version.js

```diff
@@ -0,0 +1,19 @@
+// System Objects
+// ...
+
+// Third Party Dependencies
+// ...
+
+// Internal
+var Tessel = require('./tessel');
+
+/*
+  Gathers node version.
+*/
+Tessel.prototype.fetchCurrentNodeVersion = function() {
+  return this.simpleExec(['node', '--version'])
+    .then(function versionRead(version) {
```

Comment:
Okay. I was following what I saw, and it was a lot of mixed between `function` keyword and arrow functions.